swarm/src/handler.rs

```diff
@@ -224,6 +277,76 @@ pub trait ConnectionHandler: Send + 'static {
     {
         ConnectionHandlerSelect::new(self, other)
     }
+
+    /// informs the handler about an event coming from the outside in the handler.
```

Comment:
```suggestion
    /// Informs the handler about an event from the [`NetworkBehaviour`].
```

I would find this easier to understand. What do you think?

Reply:
yeah I agree, thanks!